server/jobs/onboarding/email.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-param-reassign */
+import { developerEmailTransport, opsEmailTransport } from '../../modules/email/emailModule';
+/**
+ * @desc Automates developer onboarding on email
+ *
+ * @param {array} placement Placement record whose developer is to be offboarded
+ * @param {object} automationResult Result of automation job
+ * @returns {void}
+ */
+const emailOnboarding = async (placement, automationResult) => {
+  const { fellow } = placement;
+  const { client_name: partnerName } = placement;
+  const { client_location: partnerLocation } = placement;
+  const { start_date: startDate } = placement;
```

Comment:
You can combine all the destructuring above into a single statement like this:
```
const {
    fellow, client_name: partnerName, client_location: partnerLocation, start_date: startDate,
  } = placement;
```

Reply:
I have implemented it